mapreduce/README.md

```diff
@@ -1,32 +1,81 @@
-# Mapreduce Integration
+# Mapreduce Check
 
-## Overview
+# Overview
 
-Get metrics from mapreduce service in real time to:
+The Hadoop Mapreduce check lets you monitor the status and duration of map and reduce tasks.
 
-* Visualize and monitor mapreduce states
-* Be notified about mapreduce failovers and events.
+To collect other Hadoop-related metrics, see the hdfs_datanode, hdfs_namenode, and yarn checks.
 
-## Installation
+# Installation
 
-Install the `dd-check-mapreduce` package manually or with your favorite configuration manager
+The Agent's Mapreduce check is packaged with the Agent, so simply [install the Agent](https://app.datadoghq.com/account/settings#agent) on your Hadoop cluster's master nodes. If you need the newest version of the check, install the `dd-check-mapreduce` package.
 
-## Configuration
+# Configuration
 
-Edit the `mapreduce.yaml` file to point to your server and port, set the masters to monitor
+Create a file `mapreduce.yaml` in the Agent's `conf.d` directory:
 
-## Validation
+```
+instances:
+  # The Mapreduce check collects metrics from YARN's ResourceManager. Run the 
+  # check on your master node and specify the ResourceManager URI below.
+  # If you don't know your ResourceManager's port, you can find it in the 
+  # yarn-site.xml conf file under the property yarn.resourcemanager.webapp.address
+  - resourcemanager_uri: http://localhost:8088
 
-When you run `datadog-agent info` you should see something like the following:
+init_config:
+ general_counters:
+ - counter_group_name: 'org.apache.hadoop.mapreduce.TaskCounter'
+   counters:
+   - counter_name: 'MAP_INPUT_RECORDS'
+   - counter_name: 'MAP_OUTPUT_RECORDS'
+   - counter_name: 'REDUCE_INPUT_RECORDS'
+   - counter_name: 'REDUCE_OUTPUT_RECORDS'
+# Add more counters
+# - counter_group_name: 'org.apache.hadoop.mapreduce.FileSystemCounter'
+#   counters:
+#     - counter_name: 'HDFS_BYTES_READ'
+# etc
+```
 
-    Checks
-    ======
+Restart the Agent to begin sending Mapreduce metrics to Datadog.
 
-        mapreduce
-        -----------
-          - instance #0 [OK]
-          - Collected 39 metrics, 0 events & 7 service checks
+# Validation
 
-## Compatibility
+Run the Agent's `info` subcommand and look for `mapreduce` under the Checks section:
 
-The mapreduce check is compatible with all major platforms
+```
+  Checks
+  ======
+    [...]
+
+    mapreduce
+    -------
+      - instance #0 [OK]
+      - Collected 36 metrics, 0 events & 1 service check
+
+    [...]
+```
+
+# Compatibility
+
+The mapreduce check is compatible with all major platforms.
+
+# Metrics
+
+See [metadata.csv](https://github.com/DataDog/integrations-core/blob/master/mapreduce/metadata.csv) for a list of metrics provided by this check.
+
+The metrics available are collected using df from Spotify’s Snakebite. hdfs.in_use is calculated by dividing used by capacity.
```

Comment:
@truthbk I'm not sure, I just copied this from the Docs site. Here's the commit (from 2015):

https://github.com/DataDog/documentation/commit/803109386

It does seem like a mistaken carryover from an older version of the check, yes? I can remove.